scripts/workflow-helper.sh

```
@@ -0,0 +1,132 @@
+case "${1:-status}" in
+    "status")
+        show_workflow_status
+        ;;
+    "tasks")
+        show_assigned_tasks
+        ;;
+    "branch")
+        echo $(get_assigned_branch)
+        ;;
+    "check")
+        local expected=$(get_assigned_branch)
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)